### PR TITLE
zh-translation:/docs/tasks/observability/distributed-tracing/lightste…

### PR DESCRIPTION
**Pull request by niceforbear**
ref: https://github.com/servicemesher/istio-official-translation/issues/<issueID>

### CODE REVIEW

**Comment by tanjunchen** [comment]
/ok-to-test
please fix conclicts.Thanks

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 1]
```diff
@@ -52,9 +49,9 @@ This task uses the [Bookinfo](/docs/examples/bookinfo/) sample application as an
         --set values.global.tracer.lightstep.cacertPath="/etc/lightstep/cacert.pem"
     {{< /text >}}
 
-1.  Store your satellite pool's certificate authority certificate as a secret in the default namespace.
-    For LightStep Tracing users, download and use [this certificate](https://docs.lightstep.com/docs/use-istio-as-your-service-mesh-with-lightstep).
-    If you deploy the Bookinfo application in a different namespace, create the secret in that namespace instead.
+1.  在 default namespace 下，存储您的 satellite 池的证书颁发机构证书作为一个 secret。
```
Comment (ilylia) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md]: namespace 可以翻译为 命名空间，下同

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 2]
```diff
@@ -76,66 +73,65 @@ This task uses the [Bookinfo](/docs/examples/bookinfo/) sample application as an
...
 {{< /warning >}}
 
-## Trace sampling
+## 跟踪采样{#trace-sampling}
 
-Istio captures traces at a configurable trace sampling percentage. To learn how to modify the trace sampling percentage,
-visit the [Distributed Tracing trace sampling section](../overview/#trace-sampling).
-When using LightStep, we do not recommend reducing the trace sampling percentage below 100%. To handle a high traffic mesh,
-consider scaling up the size of your satellite pool.
+Istio 以可配置的跟踪采样百分比来捕获 trace。
```
Comment (ilylia) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md]: 建议统一一下，trace 或者 跟踪，要么都不翻译，要么都翻译

**Comment by tanjunchen** [comment]
@niceforbear 请解决下冲突

**Comment by tanjunchen** [comment]
@niceforbear  @ilylia  @SataQiu  @rootsongjc  
Thanks

The file had been translated , so close this pr . Thanks your jobs!

**Pull request closed without merging**